bom/deployment/pom.xml

```diff
@@ -346,6 +346,11 @@
                 <artifactId>quarkus-elytron-security-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-elytron-security-properties-file-deployment</artifactId>
```

Comment:
I'm wondering if we should introduce `elytron` now? We named everything `security` before to not expose it.

Reply:
The artifacts were still called Elytron. I am ok with renaming it.

Reply:
Yeah, just saw that. I would say let's not block this PR for that. We can get it in and then decide to rename all of them consistently if we want to do so.